press/press/doctype/release_group/release_group.py

```diff
@@ -1338,6 +1339,24 @@ def set_default_delta_builds_flags(self):
 	def is_version_14_or_higher(self):
 		return frappe.get_cached_value("Frappe Version", self.version, "number") >= 14
 
+	def setup_default_feature_flags(self):
+		DETAULT_FEATURE_FLAGS = {
+			"Version 14": {"merge_default_and_short_rq_queues": True},
+			"Version 15": {
+				"gunicorn_threads_per_worker": "4",
```

Comment:
This can be made default on v14 too AFAIK

Reply:
Except ancient versions. 

As in if someone chooses to upgrade without upgrading frappe version. 

https://frappecloud.com/app/query-report/Frappe%20Version%20Rollout